setup.py

```diff
@@ -32,6 +32,9 @@
         "console_scripts": [
             "oremda = oremda.cli:main",
         ],
-        "oremda.cli.plugins": ["run = oremda.cli.run:main"],
+        "oremda.cli.plugins": [
+            "run = oremda.cli.run:main",
+            "runner = oremda.cli.runner:main",
```

Comment:
I didn't realize there is a `run` and a `runner`. What is the difference?

Reply:
One runs a pipeline on the command line and the new one is a service that interacts with the web server to execute pipelines.